src/context/XDEFIProvider/types.ts

```diff
@@ -0,0 +1,27 @@
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+
+import { AnyFunction } from '../../types/common'
+
+export interface InitialState {
+  ethereumWallet: HDWallet | null
+  xfiBitcoinProvider: IXfiBitcoinProvider | null
+  xfiLitecoinProvider: IXfiLitecoinProvider | null
+  isWalletLoading: boolean
+}
+
+export interface IXfiBitcoinProvider {
+  accounts: string[]
+  chainId: string
+  network: string
+  request: (e: { method: string; params: any[] }, cb: AnyFunction) => Promise<any>
+  signTransaction: (e: any) => Promise<void>
+  transfer: (e: any) => Promise<void>
+}
+
+export interface IXfiLitecoinProvider {
+  accounts: string[]
+  chainId: string
+  network: string
+  request: (e: { method: string; params: any[] }, cb: AnyFunction) => void
+  transfer: (e: any) => void
+}
```

Comment:
This is set up so that when a XDEFI wallet is connected then it will grab the accounts of the needed wallets, in this case `litecoin` and `bitcoin` and gather some general information about the connection in a 'provider' object. 

For bitcoin it exposes the `request`, `signTransaction`, and `transfer` methods. 
https://sdk.xdefi.io/docs/HEAD/bitcoin.html

For litecoin it exposes the `request` and `transfer` methods.
https://sdk.xdefi.io/docs/HEAD/litecoin.html

These will allow you to interact with these chains